recipe/meta.yaml

```diff
@@ -21,7 +21,8 @@ source:
     - configure.patch
 
 build:
-  number: 0
+  number: 1  # [linux]
+  number: 0  # [win or osx]
```

Comment:
@ccordoba12 that approach is fine and one can even argue that it is "more correct." We do not enforce it though b/c of the maintenance burden of following two different build numbers.

So if you want to go that route it is OK. If you do not want to do that it is also fine. As the maintainer that is your call.